.github/workflows/scripts/download-nvd.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# This script fetches NVD CVE data for a given year, verifies its integrity, and uploads it to a GCS bucket.
+
+# ------------------------ Instructions ------------------------
+# 1. Ensure you have `curl`, `gunzip`, and `gsutil` installed.
+# 2. Make this script executable: `chmod +x <script_name>.sh`
+# 3. Run this script with a year as an argument: `./<script_name>.sh 2023`
+# 4. Check the script logs for any errors. The script will stop execution if there's an error.
```

Comment:
Let's use `set -eu` to ensure undefined vars generate errors and to ensure any command that fails will stop the script.

Also, it may be worth adding `-x` to the above. It would make bash to show the commands it executes and might be interesting when debugging failures in the CI step. It might also be too much output, so I leave this as a suggestion and not a blocker.